packages/client/src/components/Card/Card.tsx

```diff
@@ -7,8 +7,12 @@ type CardProps = {
 }
 
 export const Card: FC<CardProps> = ({ children, title, className }) => {
+  let cardClass = 'card';
+  if (className) {
+    cardClass = ' ' + className;
```

Comment:
Старайся больше использовать [Template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), плюсик это уже old school :):

```suggestion
    cardClass = ` ${className}`;
```

Reply:
done